lib/cx_leaderboard/application.ex

```diff
@@ -0,0 +1,18 @@
+defmodule CxLeaderboard.Application do
```

Comment:
Could you clarify why is the application file needed? I haven't programmed elixir for about a year now, so I might be rusty on some concepts, but if this lib worked without application, why does Redis make it necessary?

Reply:
This was so we could run the redix process on boot.